Styling/events by category page navbar

turquoisemelon opened the pull request:
Added styling to eventsByCategoryPage and Navigation Bar
Updated event date to be displayed properly
Updated home page header image and gradient color

All feedbacks are welcome about styling :)

auredufour commented on src/styles/App/_events-by-category-page.scss:
```diff
@@ -0,0 +1,87 @@
+                max-height: 100%;
+                max-width: 100%;
+              }
+            }
```
careful with your indentation, try to not go more than 3 indentations inside a parent, it may give you trouble for making it responsive and may be difficult if you make any mark-up changes!
All of your element have classes, try to use the classes instead of the indentation.

auredufour commented on src/styles/App/_events-by-category-page.scss:
```diff
@@ -0,0 +1,87 @@
+.events-by-category-page {
+  > h2 {
+    font-size: 2.3rem;
```
you only have one h2, you don't need to select the direct h2 from the parent, h2 should be enough.

auredufour commented on src/styles/App/_navigation.scss:
```diff
@@ -0,0 +1,29 @@
+  top: 0;
+  left: 0;
+  z-index: 9999;
+  width: 100%;
```
shouldn't be 9999, the modal is only 400 on the z-index, and should cover the nav. Can you make it 300?

auredufour commented on src/styles/App/_events-by-category-page.scss:
```diff
@@ -0,0 +1,87 @@
+      background-color: #fff;
+      width: calc(70% - 30px);
+      height: 190px;
+      list-style: none;
```
try not to add an height, but let the elements define the height for you! it would be easier to make it responsive!

auredufour commented on src/styles/App/_events-by-category-page.scss:
```diff
@@ -0,0 +1,87 @@
+              border-top: solid 1px #eee;
+
+              > span {
+                margin-left: 20px;
```
You only have one child, you don't need the > selector

auredufour commented on src/styles/App/_navigation.scss:
```diff
@@ -0,0 +1,29 @@
+
+    .navbar-items--home {
+      > a {
+        text-decoration: none;
```
same here!

srhboo commented:
agree with the above changes @auredufour mentioned - besides that, looks great!

turquoisemelon commented:
@auredufour Addressed your comments. The only thing that needs to be fixed is the layout-break in the event card when the title is in two lines. I'll work on that in another PR.
